pkg/gatewayserver/upstream/packetbroker/packetbroker.go

```diff
@@ -139,6 +141,7 @@ func (h *Handler) ConnectGateway(ctx context.Context, ids *ttnpb.GatewayIdentifi
 			"rx_rate",
 			"status_public",
 			"tx_rate",
+			"contact_info",
```

Comment:
Basically added it in order to trigger
https://github.com/TheThingsNetwork/lorawan-stack/blob/e0f0719673c291b87b607712cf465f1179655308/pkg/packetbrokeragent/grpc_gspba.go#L161

But indeed it should be `administrative_contact` and `technical_contact`. 
Since neither the `contact_info` field mask or its value was there before I'm not adding them (specially since the field is deprecated)

Reply:
I interpret @johanstokking's comment as 'why are you making it part of the field mask, but you are not making it part of the `PacketBrokerGateway` on line 120.

Reply:
I believe the reply answers that as well, the idea of the field mask was to trigger the usage of the admin/tech fields which is wrong by itself. In addition to that the `ContactInfo` field wasn't present before on the `PacketBrokerGateway` on line 120 and since its deprecated it shouldn't be there, right?.

Reply:
The fact that it wasn't there before is a mistake in itself, as the code on the PBA side depends on the GS actually pushing the contact info. I would argue that the field is deprecated, not removed, so it should still be retrieved by the GS on connection and sent to PBA as well, since the PBA logic does use it as a failover when the explicit contacts are not present.